cursor_pr_review.py

```
@@ -24,8 +16,26 @@
 import requests
 import yaml
+from functools import wraps
+
+# Import new prompt management system
+try:
+    from prompt_manager import PromptManager
+    from prompt_cli import PromptCLI
+    PROMPT_MANAGER_AVAILABLE = True
+except ImportError:
+    PROMPT_MANAGER_AVAILABLE = False
+    logger.warning("Advanced prompt management not available")
```

Ruff (0.11.9)

Undefined name `logger`

(F821)